### ISSUE TITLE
SVG files are sligthly off in some ways

### ISSUE THREAD

**Issue by kelseasy** [ForkAwesome/Fork-Awesome#75]
As can be seen [here](https://github.com/ForkAwesome/Fork-Awesome/blob/master/src/icons/svg/github.svg) and [there](https://github.com/ForkAwesome/Fork-Awesome/blob/master/src/icons/svg/ambulance.svg) for example. The paths themselves are ok, but the cropping doesn't seem to be right.

**Comment by xuv** [comment]
Correct. This is expected somehow because of the history of the project and the way we have moved from a pure font design to a process that goes from a list of SVGs to iconfont using FontCustom.

There is a way to solve this but I have not spent the time to do this yet. Basically, adding a viewbox attribute to the SVG will solve this without influencing how the font is building. It's somehow in the back of my mind but have not had the time to add it.

**Comment by tessus** [comment]
AFAIK there are viewport attributes in the SVGs. Or am I missing something here?

**Comment by Berkmann18** [comment]
@tessus As far as I know, there's no `viewport` attributes but it can be done using `width`/`height` and `viewBox` as explained [here](http://tutorials.jenkov.com/svg/svg-viewport-view-box.html).
